cpp/src/io/parquet/reader_impl_helpers.cpp

```diff
@@ -77,14 +79,15 @@ type_id to_type_id(SchemaElement const& schema,
   parquet::Type const physical            = schema.type;
   parquet::LogicalType const logical_type = schema.logical_type;
   parquet::ConvertedType converted_type   = schema.converted_type;
-  int32_t decimal_scale                   = schema.decimal_scale;
+  int32_t decimal_precision               = schema.decimal_precision;
 
   // Logical type used for actual data interpretation; the legacy converted type
   // is superceded by 'logical' type whenever available.
   auto const inferred_converted_type = logical_type_to_converted_type(logical_type);
-  if (inferred_converted_type != parquet::UNKNOWN) converted_type = inferred_converted_type;
-  if (inferred_converted_type == parquet::DECIMAL && decimal_scale == 0)
-    decimal_scale = schema.logical_type.DECIMAL.scale;
+  if (inferred_converted_type != parquet::UNKNOWN) { converted_type = inferred_converted_type; }
+  if (inferred_converted_type == parquet::DECIMAL) {
+    decimal_precision = schema.logical_type.DECIMAL.precision;
+  }
```

Comment:
precision not scale + clang-format

Reply:
I'm using pre-commit hooks so the code should have been formatted before committing.

Reply:
Sorry, this was just a comment as to what had changed, nothing to do here.